internal/sanitize.go

```diff
@@ -24,6 +24,15 @@ const labelKeySizeLimit = 100
 // Sanitize returns a string that is trunacated to 100 characters if it's too
 // long, and replaces non-alphanumeric characters to underscores.
 func Sanitize(s string) string {
+	if len(s) == 0 {
+		return s
```

Comment:
Should we also check empty string in stackdriver.go? I don't think Stackdriver can take empty as label key.

Reply:
We should check that at tag.NewKey. Validation in the tag package is not very complete, I am going to work on that later today.

Reply:
@songy23 you might be interested in seeing this issue https://github.com/census-instrumentation/opencensus-go/issues/171 in relation to empty label keys.

Reply:
I see, thanks.